OpenRA.Mods.Common/Widgets/WorldCommandWidget.cs

```diff
@@ -13,15 +13,13 @@
 using System.Drawing;
 using System.Linq;
 using OpenRA.Graphics;
-using OpenRA.Mods.Common.Orders;
 using OpenRA.Mods.Common.Traits;
-using OpenRA.Orders;
-using OpenRA.Primitives;
 using OpenRA.Traits;
 using OpenRA.Widgets;
 
 namespace OpenRA.Mods.Common.Widgets
 {
+	/// <summary> Contains all functions that are valid for players and observers/spectators. </summary>
 	public class WorldCommandWidget : Widget
```

Comment:
Probably not appropriate for this PR, but the junk that's left in here really belongs in a `ChromeLogic` class instead of a widget.